src/boxplot.jl

```diff
@@ -21,10 +22,20 @@ notch_width(q2, q4, N) = 1.58 * (q4-q2)/sqrt(N)
     bw == nothing && (bw = 0.8)
     @assert whisker_width == :match || whisker_width >= 0 "whisker_width must be :match or a positive number"
     ww = whisker_width == :match ? bw : whisker_width
+
+    ngroups = length(unique(plotattributes[:group]))
+    thisgroup = plotattributes[:series_plotindex]
+
+    outercenter = 0.5 * bw - bw / 2ngroups
+    offsets = collect(-outercenter:(bw / ngroups):outercenter)
+    bw = bw / ngroups
```

Comment:
Maybe consider:

```julia
bw = bw / ngroups * 0.9
ww = ww / ngroups * 0.9
```

to allow for some space within boxes of the same group.

Reply:
I think having the bars touching (as in `groupedbar`) makes sense - I was just wondering if there would be an appropriate keyword to grab to adjust that.

FWIW, this is what your suggestion looks like:

![screen shot 2018-10-15 at 15 05 10](https://user-images.githubusercontent.com/3502975/46972117-c93c3d80-d08b-11e8-94c1-512ba9b54431.png)
![screen shot 2018-10-15 at 15 05 00](https://user-images.githubusercontent.com/3502975/46972128-ce00f180-d08b-11e8-8542-894d905986ef.png)

Reply:
The last thing is whether we should get an attribute to avoid "touching" boxplots - and what the default should be. Maybe they can default with no distance and `boxplot(...., space = 0.1)` could create some space between them (like in the example above, with `bw = bw / ngroups * (1-space)`.

@mkborregaard: what do you think is a sensible name for the attribute and a sensible default?

Reply:
`spacing`? I think sth like 0.1 should be the default.